cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -674,18 +674,15 @@ public static String generatePermissionString(int permissions) {
     return permissionStringBuilder.toString();
   }
 
-  private void extractArchive(Path file, Path targetDir, Function<InputStream, ArchiveInputStream> unpacker) {
+  private void extractTarArchive(Path file, Path targetDir, Function<InputStream, ArchiveInputStream> unpacker) {
 
     this.context.trace("Unpacking archive {} to {}", file, targetDir);
     try (InputStream is = Files.newInputStream(file); ArchiveInputStream ais = unpacker.apply(is)) {
```

Comment:
Shouldn't we use try-with-resources as before to have automatic resource management or is there a reason for manually closing?

Reply:
I didn't know of this concept, thanks for pointing it out. Fixed 👍